openid-4-verifiable-presentations-1_0.md

```diff
@@ -1856,6 +1856,19 @@ Requests from the Wallet to the Verifier SHOULD be sent with the minimal amount
 
 In the event that another component is invoked instead of the Wallet, the End-User MUST be informed and give consent before the invoked component returns the `wallet_unavailable` Authorization Error Response to the Verifier.
 
+## Digital Credential API Error Responses {#privacy-dc-api-error}
+
+Returning any protocol error, regardless of content, may reveal additional information about the user’s underlying credentials in a way that is unique to the Digital Credentials API. For an improved user experience, platform implementations only allow wallets to be selected that satisfy the request. Protocol error responses can only be returned by a Wallet and so always reveal that the user is in possession of credentials that satisfy the request. This is in contrast to other engagement methods, where the Wallet receives the request before learning if it can be fulfilled. What is revealed by a wallet in those cases depends on how each individual wallet processes a request.
+
+The more narrow a request is, the more information is revealed. Some examples: 
+
+ * A request that can be fulfilled by a broad range of documents. This will reveal only that the user has a credential from a large set of documents..
+ * A request that requests a single document type. This will reveal the user is in possession of that credential. How sensitive this is would depend on the particular credential.
```

Comment:
```suggestion
 * A request for a single document type. This will reveal the user is in possession of that credential. How sensitive this is would depend on the particular credential.
```